lib/cocoapods/generator/app_target_helper.rb

```diff
@@ -90,10 +90,34 @@ def self.add_empty_swift_file(project, target, name = 'App')
       #
       def self.add_app_host_main_file(project, target, platform, name = 'App')
         source_file = AppTargetHelper.create_app_host_main_file(project, platform, name)
-        source_file_ref = project.new_group(name, name).new_file(source_file)
+        group = project[name] || project.new_group(name, name)
+        source_file_ref = group.new_file(source_file)
         target.add_file_references([source_file_ref])
       end
 
+      # Creates a default launchscreen storyboard.
+      #
+      # @param  [Project] project
+      #         the Xcodeproj to generate the launchscreen storyboard into.
+      #
+      # @param  [PBXNativeTarget] target
+      #         the native target to link the generated launchscreen storyboard into.
+      #
+      # @param  [Symbol] platform
+      #         the platform of the target. Can be `:ios` or `:osx`, etc.
+      #
+      # @param  [String] name
+      #         The name to use for the target, defaults to 'App'.
+      #
+      # @return [PBXFileReference] the created file reference of the launchscreen storyboard.
+      #
+      def self.add_launchscreen_storyboard(project, target, name = 'App')
+        launch_storyboard_file = AppTargetHelper.create_launchscreen_storyboard_file(project, name)
```

Comment:
This doesn't need to reference `AppTargetHelper` directly

Reply:
ah the other ones do, I copied pasted. Meh